compiler/rustc_mir_transform/src/post_analysis_normalize.rs

```diff
@@ -1,26 +1,27 @@
-//! Normalizes MIR in RevealAll mode.
+//! Normalizes MIR in TypingMode::PostAnalysis mode, most notably revealing
+//! its opaques.
```

Comment:
Also maybe worth noting that we normalize non-final projections that aren't further specializable or whatever.